KubeArmor/core/containerdHandler.go

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2021 Authors of KubeArmor
 
+// Package core declaration
```

Comment:
```suggestion
// Package core is responsible for initiating and maintaining interactions between external entities like K8s,CRIs and internal KubeArmor entities like eBPF Monitor and Log Feeders
```

Let's move this comment declaration to `core/kubeArmor.go`